services_registry/ui.py

```diff
@@ -59,10 +59,11 @@ def explore_service(name, url, info, error):
             "error": error,
             "url": url
         }
-
-    info = info.get('response',{}).get('results',{})
+    info = info.get('response')
+    if info.get('results'):
```

Comment:
Wot iz dis ??????

Reply:
JSON formats not matching with new Beacons and old Beacons so I made this quick fix to adapting the new Beacons JSON format.

Reply:
`info.get('results')` is a search. You test the result to check if there is anything (be it None, {}, [], 0, or False).
If this is not the case, you search again _for the same thing_ and you say "If I don't find it, I'll make it None", but you just tested that you found something.

In other words, the comment doesn't help me understand more the logic behind.
Let's start from the beginning: What was wrong with the previous code? What does it miss to capture?

Reply:
this is the old format: https://playground.rd-connect.eu/beacon2/api
this is the new format: https://progenetix.org/beacon/info

In the old format, the fields that I need are under the response -> results
In the new format, the fields that I need are under the response.

Reply:
ok, then you can code it a bit differently to avoid the double hashtable search.
Side question: why do you want to support the old format?

Reply:
3 of them have the old format and one of them has the new format. So I need to support two formats at least all of them have the same format.